src/init.jl

```diff
@@ -1,5 +1,6 @@
 # PyPlot initialization — the hardest part is finding a working backend.
 using VersionParsing
+using Compat
```

Comment:
This line is not needed — Compat is already imported in PyPlot.jl, from which `init.jl` is included.